enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,300 @@
+```
+All label values would be rounded up to the nearest step size, so if it took 4 minutes to complete bootstrapping and the agreed upon step size was 2.5 minutes, the value would be rounded to 5.
+
+The proposed periods to track are:
+* `infrastructure`
+* `bootstrap`
+* `api`
+* `operators`
+
+One sample would be pushed per command invocation.
+
+#### cluster_installation_modification
+The `cluster_installation_modification` metric represents whether a user modified an installer created asset prior to running the `create` command.
+
+Feedback would be appreciated on two potential designs. One which aggregates the values into a single sample and one that pushes multiple samples:
+
+```
+# HELP cluster_installation_modification lists all the assets that were changed from the original value by the user before the create command was called.
+# TYPE cluster_installation_modification counter
+cluster_installation_modification{asset="cluster-config.yaml,cluster-scheduler-02-config.yml"} 1
+```
+In this case, the values would be sorted in order to reduce cardinality. The benefit of this approach is that it creates a correlation between the modified resources in a particular run. It is less natural than the following approach and could make querying more difficult (feedback on querying, please).
+
+```
+# HELP cluster_installation_modification lists all the assets that were changed from the original value by the user before the create command was called.
+# TYPE cluster_installation_modification counter
+cluster_installation_modification{asset="cluster-config.yaml"} 1
+cluster_installation_modification{asset="cluster-scheduler-02-config.yml"} 1
+```
+The potential values for these labels would be the ignition files: `bootstrap.ign`, `master.ign`, or `worker.ign` or the manifest files:
+```
+04-openshift-machine-config-operator.yaml  cluster-network-01-crd.yml       etcd-client-secret.yaml                etcd-namespace.yaml                 machine-config-server-tls-secret.yaml
+cloud-provider-config.yaml                 cluster-network-02-config.yml    etcd-host-service-endpoints.yaml       etcd-service.yaml                   openshift-config-secret-pull-secret.yaml
+cluster-config.yaml                        cluster-proxy-01-config.yaml     etcd-host-service.yaml                 etcd-serving-ca-configmap.yaml
+cluster-dns-02-config.yml                  cluster-scheduler-02-config.yml  etcd-metric-client-secret.yaml         etcd-signer-secret.yaml
+cluster-infrastructure-02-config.yml       cvo-overrides.yaml               etcd-metric-serving-ca-configmap.yaml  kube-cloud-config.yaml
+cluster-ingress-02-config.yml              etcd-ca-bundle-configmap.yaml    etcd-metric-signer-secret.yaml         kube-system-configmap-root-ca.yaml
+```
+
+The number of potential values is approximately 30.
+
+Either of these proposed metrics would be pushed only upon successful completion of the `create cluster` command.
+
+#### Metric Labels
```

Comment:
For every metric/label combinations that are proposed here, I would like to see a back of the envelope cardinality calculation. Even reading through this enhancement I have only a vague idea of the numbers. Especially for the back end side of things, capacity planning etc. this is very important.